apps/condo/domains/organization/components/OrganizationSelect.tsx

```diff
@@ -46,11 +46,9 @@ const blackSelectCss = css`
 `
 // TODO(zuch): can't use emotion css here
 const optionStyle: React.CSSProperties = {
-    fontSize: '16px',
-    lineHeight: '24px',
+    fontSize: '14px',
```

Comment:
Have you discussed this with Max?
![image](https://user-images.githubusercontent.com/1640424/125054919-22f8b780-e0c0-11eb-89d5-2c4d9a072b6d.png)

Reply:
yes